src/Symfony/Component/HttpClient/Response/StreamWrapper.php

```diff
@@ -178,6 +178,56 @@ public function stream_read(int $count)
         return '';
     }
 
+    public function stream_set_option(int $option, int $arg1, ?int $arg2): bool
+    {
+        if (null === $this->handle || 'stream' !== get_resource_type($this->handle)) {
+            trigger_error(sprintf('The "$handle" property of "%s" need to be a stream.', __CLASS__), E_USER_WARNING);
```

Comment:
null handler cannot occur at this point (it is when the stream is not open but you cannot call that kind of function if you don't have an openned stream?!)

Reply:
`$handle` is null when `CurlHttpClient` is used, as [Nicolas commented](https://github.com/symfony/symfony/issues/34943#issuecomment-564981150) :)

Reply:
s/need/needs
but since the response is set in a property, I think it would be nicer to mention it
btw, how does PHP behave when we return false? does it throw another warning on its own?